src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,7 @@
+@mixin animation-hover($property, $value) {
+  transition: $property $time-for-animation;
```

Comment:
The variable `$time-for-animation` is used but not defined in this mixin. Make sure to either define it within the mixin or pass it as a parameter to ensure the mixin is functional.